<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>814</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>120</y>
     <width>291</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="standardDeviation_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard Deviation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mean_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boxPlot_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Box plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="varience_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Varience</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_z_zcore2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 8pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>Z_Score for any value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yVariableInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="Enter value"/>
    </property>
   </widget>
   <widget class="QPushButton" name="histogram_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="vector1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>vector2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pieChart_2_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pie Chart</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>291</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="vector1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>vector1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mean_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="varience_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Varience</string>
    </property>
   </widget>
   <widget class="QPushButton" name="standardDeviation_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Standard Deviation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xVariableInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_z_zcore">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 8pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>Z_Score for any value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boxPlot_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Box plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="histogram_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pieChart_1_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pie Chart</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="barchart">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Bar chart</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableOfItems">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>710</y>
     <width>1131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="correlationC_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Correlation Coefficient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>540</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="expectedValue_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Enter value to expect it from linear regression</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expectedVariableInput">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="result">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="documentTitle">
    <string comment="Welcome" extracomment="Welcome"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string comment="Welcome" extracomment="Welcome"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Scatterplot_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Scatter Plot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
